[2016-12-07 07:51:14] request.INFO: Matched route "fos_user_registration_register". {"route":"fos_user_registration_register","route_parameters":{"_controller":"FOS\\UserBundle\\Controller\\RegistrationController::registerAction","_route":"fos_user_registration_register"},"request_uri":"http://localhost:8080/test_symfo3/web/register/","method":"POST"} []
[2016-12-07 07:51:14] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2016-12-07 07:51:15] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\TableNotFoundException: "An exception occurred while executing 'SELECT t0.username AS username_1, t0.username_canonical AS username_canonical_2, t0.email AS email_3, t0.email_canonical AS email_canonical_4, t0.enabled AS enabled_5, t0.salt AS salt_6, t0.password AS password_7, t0.last_login AS last_login_8, t0.confirmation_token AS confirmation_token_9, t0.password_requested_at AS password_requested_at_10, t0.roles AS roles_11, t0.id AS id_12 FROM fos_user t0 WHERE t0.username_canonical = ?' with params ["test"]:  SQLSTATE[42S02]: Base table or view not found: 1146 Table 'test_symfo3.fos_user' doesn't exist" at C:\xampp\htdocs\test_symfo3\vendor\doctrine\dbal\lib\Doctrine\DBAL\Driver\AbstractMySQLDriver.php line 53 {"exception":"[object] (Doctrine\\DBAL\\Exception\\TableNotFoundException(code: 0): An exception occurred while executing 'SELECT t0.username AS username_1, t0.username_canonical AS username_canonical_2, t0.email AS email_3, t0.email_canonical AS email_canonical_4, t0.enabled AS enabled_5, t0.salt AS salt_6, t0.password AS password_7, t0.last_login AS last_login_8, t0.confirmation_token AS confirmation_token_9, t0.password_requested_at AS password_requested_at_10, t0.roles AS roles_11, t0.id AS id_12 FROM fos_user t0 WHERE t0.username_canonical = ?' with params [\"test\"]:\n\nSQLSTATE[42S02]: Base table or view not found: 1146 Table 'test_symfo3.fos_user' doesn't exist at C:\\xampp\\htdocs\\test_symfo3\\vendor\\doctrine\\dbal\\lib\\Doctrine\\DBAL\\Driver\\AbstractMySQLDriver.php:53, Doctrine\\DBAL\\Driver\\PDOException(code: 42S02): SQLSTATE[42S02]: Base table or view not found: 1146 Table 'test_symfo3.fos_user' doesn't exist at C:\\xampp\\htdocs\\test_symfo3\\vendor\\doctrine\\dbal\\lib\\Doctrine\\DBAL\\Driver\\PDOStatement.php:93, PDOException(code: 42S02): SQLSTATE[42S02]: Base table or view not found: 1146 Table 'test_symfo3.fos_user' doesn't exist at C:\\xampp\\htdocs\\test_symfo3\\vendor\\doctrine\\dbal\\lib\\Doctrine\\DBAL\\Driver\\PDOStatement.php:91)"} []
[2016-12-07 08:13:35] request.INFO: Matched route "homepage". {"route":"homepage","route_parameters":{"_controller":"AppBundle\\Controller\\DefaultController::indexAction","_route":"homepage"},"request_uri":"http://localhost:8080/test_symfo3/web/","method":"GET"} []
[2016-12-07 08:13:36] security.DEBUG: Read existing security token from the session. {"key":"_security_main"} []
[2016-12-07 08:13:36] security.DEBUG: User was reloaded from a user provider. {"username":"test","provider":"FOS\\UserBundle\\Security\\UserProvider"} []
[2016-12-07 08:13:36] request.CRITICAL: Uncaught PHP Exception InvalidArgumentException: "Unable to find template "test_symfo3/web/template/inedx.html.twig" (looked into: C:\xampp\htdocs\test_symfo3\app/Resources/views, C:\xampp\htdocs\test_symfo3\vendor\symfony\symfony\src\Symfony\Bridge\Twig/Resources/views/Form)." at C:\xampp\htdocs\test_symfo3\vendor\symfony\symfony\src\Symfony\Bridge\Twig\TwigEngine.php line 128 {"exception":"[object] (InvalidArgumentException(code: 0): Unable to find template \"test_symfo3/web/template/inedx.html.twig\" (looked into: C:\\xampp\\htdocs\\test_symfo3\\app/Resources/views, C:\\xampp\\htdocs\\test_symfo3\\vendor\\symfony\\symfony\\src\\Symfony\\Bridge\\Twig/Resources/views/Form). at C:\\xampp\\htdocs\\test_symfo3\\vendor\\symfony\\symfony\\src\\Symfony\\Bridge\\Twig\\TwigEngine.php:128, Twig_Error_Loader(code: 0): Unable to find template \"test_symfo3/web/template/inedx.html.twig\" (looked into: C:\\xampp\\htdocs\\test_symfo3\\app/Resources/views, C:\\xampp\\htdocs\\test_symfo3\\vendor\\symfony\\symfony\\src\\Symfony\\Bridge\\Twig/Resources/views/Form). at C:\\xampp\\htdocs\\test_symfo3\\vendor\\twig\\twig\\lib\\Twig\\Loader\\Filesystem.php:247)"} []
[2016-12-07 08:13:36] security.DEBUG: Stored the security token in the session. {"key":"_security_main"} []
[2016-12-07 08:13:38] request.INFO: Matched route "homepage". {"route":"homepage","route_parameters":{"_controller":"AppBundle\\Controller\\DefaultController::indexAction","_route":"homepage"},"request_uri":"http://localhost:8080/test_symfo3/web/","method":"GET"} []
[2016-12-07 08:13:38] security.DEBUG: Read existing security token from the session. {"key":"_security_main"} []
[2016-12-07 08:13:38] security.DEBUG: User was reloaded from a user provider. {"username":"test","provider":"FOS\\UserBundle\\Security\\UserProvider"} []
[2016-12-07 08:13:38] request.CRITICAL: Uncaught PHP Exception InvalidArgumentException: "Unable to find template "test_symfo3/web/template/inedx.html.twig" (looked into: C:\xampp\htdocs\test_symfo3\app/Resources/views, C:\xampp\htdocs\test_symfo3\vendor\symfony\symfony\src\Symfony\Bridge\Twig/Resources/views/Form)." at C:\xampp\htdocs\test_symfo3\vendor\symfony\symfony\src\Symfony\Bridge\Twig\TwigEngine.php line 128 {"exception":"[object] (InvalidArgumentException(code: 0): Unable to find template \"test_symfo3/web/template/inedx.html.twig\" (looked into: C:\\xampp\\htdocs\\test_symfo3\\app/Resources/views, C:\\xampp\\htdocs\\test_symfo3\\vendor\\symfony\\symfony\\src\\Symfony\\Bridge\\Twig/Resources/views/Form). at C:\\xampp\\htdocs\\test_symfo3\\vendor\\symfony\\symfony\\src\\Symfony\\Bridge\\Twig\\TwigEngine.php:128, Twig_Error_Loader(code: 0): Unable to find template \"test_symfo3/web/template/inedx.html.twig\" (looked into: C:\\xampp\\htdocs\\test_symfo3\\app/Resources/views, C:\\xampp\\htdocs\\test_symfo3\\vendor\\symfony\\symfony\\src\\Symfony\\Bridge\\Twig/Resources/views/Form). at C:\\xampp\\htdocs\\test_symfo3\\vendor\\twig\\twig\\lib\\Twig\\Loader\\Filesystem.php:247)"} []
[2016-12-07 08:13:38] security.DEBUG: Stored the security token in the session. {"key":"_security_main"} []
[2016-12-07 08:13:55] request.INFO: Matched route "homepage". {"route":"homepage","route_parameters":{"_controller":"AppBundle\\Controller\\DefaultController::indexAction","_route":"homepage"},"request_uri":"http://localhost:8080/test_symfo3/web/","method":"GET"} []
[2016-12-07 08:13:55] security.DEBUG: Read existing security token from the session. {"key":"_security_main"} []
[2016-12-07 08:13:55] security.DEBUG: User was reloaded from a user provider. {"username":"test","provider":"FOS\\UserBundle\\Security\\UserProvider"} []
[2016-12-07 08:13:55] request.CRITICAL: Uncaught PHP Exception InvalidArgumentException: "Unable to find template "test_symfo3/web/template/inedx.html.twig" (looked into: C:\xampp\htdocs\test_symfo3\app/Resources/views, C:\xampp\htdocs\test_symfo3\vendor\symfony\symfony\src\Symfony\Bridge\Twig/Resources/views/Form)." at C:\xampp\htdocs\test_symfo3\vendor\symfony\symfony\src\Symfony\Bridge\Twig\TwigEngine.php line 128 {"exception":"[object] (InvalidArgumentException(code: 0): Unable to find template \"test_symfo3/web/template/inedx.html.twig\" (looked into: C:\\xampp\\htdocs\\test_symfo3\\app/Resources/views, C:\\xampp\\htdocs\\test_symfo3\\vendor\\symfony\\symfony\\src\\Symfony\\Bridge\\Twig/Resources/views/Form). at C:\\xampp\\htdocs\\test_symfo3\\vendor\\symfony\\symfony\\src\\Symfony\\Bridge\\Twig\\TwigEngine.php:128, Twig_Error_Loader(code: 0): Unable to find template \"test_symfo3/web/template/inedx.html.twig\" (looked into: C:\\xampp\\htdocs\\test_symfo3\\app/Resources/views, C:\\xampp\\htdocs\\test_symfo3\\vendor\\symfony\\symfony\\src\\Symfony\\Bridge\\Twig/Resources/views/Form). at C:\\xampp\\htdocs\\test_symfo3\\vendor\\twig\\twig\\lib\\Twig\\Loader\\Filesystem.php:247)"} []
[2016-12-07 08:13:55] security.DEBUG: Stored the security token in the session. {"key":"_security_main"} []
[2016-12-07 08:14:03] request.INFO: Matched route "homepage". {"route":"homepage","route_parameters":{"_controller":"AppBundle\\Controller\\DefaultController::indexAction","_route":"homepage"},"request_uri":"http://localhost:8080/test_symfo3/web/","method":"GET"} []
[2016-12-07 08:14:03] security.DEBUG: Read existing security token from the session. {"key":"_security_main"} []
[2016-12-07 08:14:03] security.DEBUG: User was reloaded from a user provider. {"username":"test","provider":"FOS\\UserBundle\\Security\\UserProvider"} []
[2016-12-07 08:14:03] request.CRITICAL: Uncaught PHP Exception InvalidArgumentException: "Unable to find template "test_symfo3/web/template/inedx.html.twig" (looked into: C:\xampp\htdocs\test_symfo3\app/Resources/views, C:\xampp\htdocs\test_symfo3\vendor\symfony\symfony\src\Symfony\Bridge\Twig/Resources/views/Form)." at C:\xampp\htdocs\test_symfo3\vendor\symfony\symfony\src\Symfony\Bridge\Twig\TwigEngine.php line 128 {"exception":"[object] (InvalidArgumentException(code: 0): Unable to find template \"test_symfo3/web/template/inedx.html.twig\" (looked into: C:\\xampp\\htdocs\\test_symfo3\\app/Resources/views, C:\\xampp\\htdocs\\test_symfo3\\vendor\\symfony\\symfony\\src\\Symfony\\Bridge\\Twig/Resources/views/Form). at C:\\xampp\\htdocs\\test_symfo3\\vendor\\symfony\\symfony\\src\\Symfony\\Bridge\\Twig\\TwigEngine.php:128, Twig_Error_Loader(code: 0): Unable to find template \"test_symfo3/web/template/inedx.html.twig\" (looked into: C:\\xampp\\htdocs\\test_symfo3\\app/Resources/views, C:\\xampp\\htdocs\\test_symfo3\\vendor\\symfony\\symfony\\src\\Symfony\\Bridge\\Twig/Resources/views/Form). at C:\\xampp\\htdocs\\test_symfo3\\vendor\\twig\\twig\\lib\\Twig\\Loader\\Filesystem.php:247)"} []
[2016-12-07 08:14:03] security.DEBUG: Stored the security token in the session. {"key":"_security_main"} []
[2016-12-07 08:14:06] request.INFO: Matched route "homepage". {"route":"homepage","route_parameters":{"_controller":"AppBundle\\Controller\\DefaultController::indexAction","_route":"homepage"},"request_uri":"http://localhost:8080/test_symfo3/web/","method":"GET"} []
[2016-12-07 08:14:06] security.DEBUG: Read existing security token from the session. {"key":"_security_main"} []
[2016-12-07 08:14:06] security.DEBUG: User was reloaded from a user provider. {"username":"test","provider":"FOS\\UserBundle\\Security\\UserProvider"} []
[2016-12-07 08:14:06] request.CRITICAL: Uncaught PHP Exception InvalidArgumentException: "Unable to find template "test_symfo3/web/template/inedx.html.twig" (looked into: C:\xampp\htdocs\test_symfo3\app/Resources/views, C:\xampp\htdocs\test_symfo3\vendor\symfony\symfony\src\Symfony\Bridge\Twig/Resources/views/Form)." at C:\xampp\htdocs\test_symfo3\vendor\symfony\symfony\src\Symfony\Bridge\Twig\TwigEngine.php line 128 {"exception":"[object] (InvalidArgumentException(code: 0): Unable to find template \"test_symfo3/web/template/inedx.html.twig\" (looked into: C:\\xampp\\htdocs\\test_symfo3\\app/Resources/views, C:\\xampp\\htdocs\\test_symfo3\\vendor\\symfony\\symfony\\src\\Symfony\\Bridge\\Twig/Resources/views/Form). at C:\\xampp\\htdocs\\test_symfo3\\vendor\\symfony\\symfony\\src\\Symfony\\Bridge\\Twig\\TwigEngine.php:128, Twig_Error_Loader(code: 0): Unable to find template \"test_symfo3/web/template/inedx.html.twig\" (looked into: C:\\xampp\\htdocs\\test_symfo3\\app/Resources/views, C:\\xampp\\htdocs\\test_symfo3\\vendor\\symfony\\symfony\\src\\Symfony\\Bridge\\Twig/Resources/views/Form). at C:\\xampp\\htdocs\\test_symfo3\\vendor\\twig\\twig\\lib\\Twig\\Loader\\Filesystem.php:247)"} []
[2016-12-07 08:14:06] security.DEBUG: Stored the security token in the session. {"key":"_security_main"} []
[2016-12-14 15:01:53] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /uploads/img/1.JPG" (from "http://localhost:8080/test_symfo3/web/app_dev.php/blog/show/21")" at C:\xampp\htdocs\test_symfo3\var\cache\prod\classes.php line 4595 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /uploads/img/1.JPG\" (from \"http://localhost:8080/test_symfo3/web/app_dev.php/blog/show/21\") at C:\\xampp\\htdocs\\test_symfo3\\var\\cache\\prod\\classes.php:4595, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\xampp\\htdocs\\test_symfo3\\var\\cache\\prod\\appProdProjectContainerUrlMatcher.php:230)"} []
[2016-12-14 15:02:08] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /uploads/img/2.JPG" (from "http://localhost:8080/test_symfo3/web/app_dev.php/blog/show/20")" at C:\xampp\htdocs\test_symfo3\var\cache\prod\classes.php line 4595 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /uploads/img/2.JPG\" (from \"http://localhost:8080/test_symfo3/web/app_dev.php/blog/show/20\") at C:\\xampp\\htdocs\\test_symfo3\\var\\cache\\prod\\classes.php:4595, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\xampp\\htdocs\\test_symfo3\\var\\cache\\prod\\appProdProjectContainerUrlMatcher.php:230)"} []
[2016-12-14 15:02:52] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /uploads/img/2.JPG" (from "http://localhost:8080/test_symfo3/web/app_dev.php/blog/show/20")" at C:\xampp\htdocs\test_symfo3\var\cache\prod\classes.php line 4595 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /uploads/img/2.JPG\" (from \"http://localhost:8080/test_symfo3/web/app_dev.php/blog/show/20\") at C:\\xampp\\htdocs\\test_symfo3\\var\\cache\\prod\\classes.php:4595, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\xampp\\htdocs\\test_symfo3\\var\\cache\\prod\\appProdProjectContainerUrlMatcher.php:230)"} []
[2016-12-14 15:03:32] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /uploads/img/2.JPG" (from "http://localhost:8080/test_symfo3/web/app_dev.php/blog/show/20")" at C:\xampp\htdocs\test_symfo3\var\cache\prod\classes.php line 4595 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /uploads/img/2.JPG\" (from \"http://localhost:8080/test_symfo3/web/app_dev.php/blog/show/20\") at C:\\xampp\\htdocs\\test_symfo3\\var\\cache\\prod\\classes.php:4595, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\xampp\\htdocs\\test_symfo3\\var\\cache\\prod\\appProdProjectContainerUrlMatcher.php:230)"} []
[2016-12-14 15:03:41] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /app/uploads/img/2.JPG" (from "http://localhost:8080/test_symfo3/web/app_dev.php/blog/show/20")" at C:\xampp\htdocs\test_symfo3\var\cache\prod\classes.php line 4595 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /app/uploads/img/2.JPG\" (from \"http://localhost:8080/test_symfo3/web/app_dev.php/blog/show/20\") at C:\\xampp\\htdocs\\test_symfo3\\var\\cache\\prod\\classes.php:4595, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\xampp\\htdocs\\test_symfo3\\var\\cache\\prod\\appProdProjectContainerUrlMatcher.php:230)"} []
[2016-12-14 15:04:25] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /app/uploads/img/2.JPG" (from "http://localhost:8080/test_symfo3/web/app_dev.php/blog/show/20")" at C:\xampp\htdocs\test_symfo3\var\cache\prod\classes.php line 4595 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /app/uploads/img/2.JPG\" (from \"http://localhost:8080/test_symfo3/web/app_dev.php/blog/show/20\") at C:\\xampp\\htdocs\\test_symfo3\\var\\cache\\prod\\classes.php:4595, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\xampp\\htdocs\\test_symfo3\\var\\cache\\prod\\appProdProjectContainerUrlMatcher.php:230)"} []
[2016-12-14 15:04:32] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /uploads/img/2.JPG" (from "http://localhost:8080/test_symfo3/web/app_dev.php/blog/show/20")" at C:\xampp\htdocs\test_symfo3\var\cache\prod\classes.php line 4595 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /uploads/img/2.JPG\" (from \"http://localhost:8080/test_symfo3/web/app_dev.php/blog/show/20\") at C:\\xampp\\htdocs\\test_symfo3\\var\\cache\\prod\\classes.php:4595, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\xampp\\htdocs\\test_symfo3\\var\\cache\\prod\\appProdProjectContainerUrlMatcher.php:230)"} []
[2016-12-14 15:04:51] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /uploads/img/2.JPG" (from "http://localhost:8080/test_symfo3/web/app_dev.php/blog/show/20")" at C:\xampp\htdocs\test_symfo3\var\cache\prod\classes.php line 4595 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /uploads/img/2.JPG\" (from \"http://localhost:8080/test_symfo3/web/app_dev.php/blog/show/20\") at C:\\xampp\\htdocs\\test_symfo3\\var\\cache\\prod\\classes.php:4595, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\xampp\\htdocs\\test_symfo3\\var\\cache\\prod\\appProdProjectContainerUrlMatcher.php:230)"} []
[2016-12-14 15:05:37] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /AppBundle/uploads/img/2.JPG" (from "http://localhost:8080/test_symfo3/web/app_dev.php/blog/show/20")" at C:\xampp\htdocs\test_symfo3\var\cache\prod\classes.php line 4595 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /AppBundle/uploads/img/2.JPG\" (from \"http://localhost:8080/test_symfo3/web/app_dev.php/blog/show/20\") at C:\\xampp\\htdocs\\test_symfo3\\var\\cache\\prod\\classes.php:4595, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\xampp\\htdocs\\test_symfo3\\var\\cache\\prod\\appProdProjectContainerUrlMatcher.php:230)"} []
[2016-12-14 15:05:40] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /AppBundle/uploads/img/2.JPG" (from "http://localhost:8080/test_symfo3/web/app_dev.php/blog/show/20")" at C:\xampp\htdocs\test_symfo3\var\cache\prod\classes.php line 4595 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /AppBundle/uploads/img/2.JPG\" (from \"http://localhost:8080/test_symfo3/web/app_dev.php/blog/show/20\") at C:\\xampp\\htdocs\\test_symfo3\\var\\cache\\prod\\classes.php:4595, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\xampp\\htdocs\\test_symfo3\\var\\cache\\prod\\appProdProjectContainerUrlMatcher.php:230)"} []
[2016-12-14 15:06:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /uploads/img/2.JPG" (from "http://localhost:8080/test_symfo3/web/app_dev.php/blog/show/20")" at C:\xampp\htdocs\test_symfo3\var\cache\prod\classes.php line 4595 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /uploads/img/2.JPG\" (from \"http://localhost:8080/test_symfo3/web/app_dev.php/blog/show/20\") at C:\\xampp\\htdocs\\test_symfo3\\var\\cache\\prod\\classes.php:4595, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\xampp\\htdocs\\test_symfo3\\var\\cache\\prod\\appProdProjectContainerUrlMatcher.php:230)"} []
